### ISSUE TITLE
Remove the DoubleTrouble class

### ISSUE THREAD

**Issue by johanley** [johanley/planisphere#23]
The idea of fusing two close stars into one object, in the context of rendering a chart for the human eye, is a valid one.

But in the current context, it makes less sense. The planisphere makes no fine distinction between magnitudes. Moreover, when the code is used to generate charts over long time scales, the fusion makes less and less sense. That's because the fusion depends on separation, and separation depends on time. For optical doubles, with independent proper motions, pairs can gradually separate over long time scales. The DoubleTrouble class is hard-coded to the present day.

Removing the DoubleTrouble class will make the task of migrating to a new data source easier. 

In short, the DoubleTrouble class, and the idea of amalgamating close doubles for this project, no longer seems to be worthwhile to me.
